src/threads.js

```diff
@@ -2817,9 +2817,9 @@ Process.prototype.reportUnicode = function (string) {
     var str = isNil(string) ? '\u0000' : string.toString();
 
     if (str.codePointAt) { // support for Unicode in newer browsers.
-        return str.codePointAt(0);
+        return str.codePointAt(0) || 0;
```

Comment:
why zero and not an empty string?

Reply:
In general, the block returns a number, so it should return a number in this case also.  And I thought line 2817 was an attempt to return zero, by creating a string with a null character in it!  So I just made that work.

Reply:
Oh wait! it's the *reportUnicode* function, duh. You're right, nevermind. Thanks!